src/api/index.ts

```diff
@@ -81,84 +75,3 @@ function createApi(context: Context): Api {
     ...tilesetsApi,
   }
 }
-
-function init(db: Database): Context {
-  // Enable auto-vacuum by setting it to incremental mode
-  // This has to be set before the anything on the db instance is called!
-  // https://www.sqlite.org/pragma.html#pragma_auto_vacuum
-  // https://www.sqlite.org/pragma.html#pragma_incremental_vacuum
-  db.pragma('auto_vacuum = INCREMENTAL')
-
-  // Enable WAL for potential performance benefits
-  // https://github.com/JoshuaWise/better-sqlite3/blob/master/docs/performance.md
-  db.pragma('journal_mode = WAL')
-
-  migrate(db, path.resolve(__dirname, '../../prisma/migrations'))
-
-  // Any import with an `active` state on startup most likely failed due to the server process stopping
-  // so we update these import records to have an error state
-  convertActiveImportsToErrorImports(db)
-
-  const piscina = new Piscina({
-    filename: path.resolve(__dirname, '../lib/mbtiles_import_worker.js'),
-    minThreads: 1,
-  })
-
-  piscina.on('error', (error) => {
-    // TODO: Do something with this error https://github.com/piscinajs/piscina#event-error
-    console.error(error)
-  })
-
-  return {
-    activeImports: new Map(),
-    db,
-    piscina,
-    upstreamRequestsManager: new UpstreamRequestsManager(),
-  }
-}
-
-const ApiPlugin: FastifyPluginAsync<MapServerOptions> = async (
-  fastify,
-  { database }
-) => {
-  if (database == null)
-    throw new Error(
-      `Instance of BetterSqlite3.Database must be specified for 'database' option (https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#class-database)`
-    )
-
-  // Create context once for each fastify instance
-  const context = init(database)
-
-  const api = createApi(context)
-
-  if (!fastify.hasDecorator('api')) {
-    fastify.decorate('api', api)
-  }
-
-  fastify.addHook('onClose', async () => {
```

Comment:
Instead of using a Fastify hook, this stuff is done in `MapServer.prototype.close`.